nixosTest matrix (prev also modularity, option docs) (#176557)
NixOS/nixpkgs: doc/stdenv/meta.chapter.md

A timeout (in seconds) for building the derivation. If the derivation takes longer than this time to build, it can fail due to breaking the timeout. However, all computers do not have the same computing power, hence some builders may decide to apply a multiplicative factor to this value. When filling this value in, try to keep it approximately consistent with other values already present in `nixpkgs`.

> This setting may be lost when the package is used as a dependency instead of being presented directly an evaluation process that handles the `meta.timeout` attribute. Meta attributes are not stored in the derivation.

Reviewer: ```suggestion
`meta` attributes are not stored in the evaluated derivation.
Therefore, this setting may be lost when the package is used as a dependency.
To be effective, it must be presented directly to an evaluation process that handles the `meta.timeout` attribute.

```
